### PR TITLE
adding bare metal walk through

### PR DESCRIPTION
**Pull request by kvahed**

### CODE REVIEW

File: docs/Manual/Tutorials/Kubernetes/bare-metal.md [hunk 1]
```diff
@@ -0,0 +1,457 @@
+# ArangoDB on bare metal Kubernetes
+
+A not of warning for lack of a better word upfront: Kubernetes is
+awesome and powerful. As with awesome and powerful things, there is
+infinite ways of setting up a k8s cluster. With great flexibility
+comes great complexity. There are inifinite ways of hitting barriers.
+
+This guide is a walk through for, again in lack of a better word,
+a reasonable and flexibel setup to get to an ArangoDB cluster setup on
+a baremetal setup.
```
Comment (lamai93) [inline docs/Manual/Tutorials/Kubernetes/bare-metal.md]:
```suggestion
a baremetal kubernetes setup.
```

File: docs/Manual/Tutorials/Kubernetes/bare-metal.md [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+
+Let there be 3 Linux boxes, `kube01`, `kube02` and `kube03`, with `kubeadm` and `kubectl` installed and off we go:
+
+* `kubeadm`, `kubectl` version `>=1.10` 
+
+## Initialise the master node
+
+The master node is outstanding in that it handles the API server and some other vital infrastructure 
+
+    kube01 > sudo kubeadm init --pod-network-cidr=10.244.0.0/16
```
Comment (lamai93) [inline docs/Manual/Tutorials/Kubernetes/bare-metal.md]: `kube0: command not found`

File: docs/Manual/Tutorials/Kubernetes/bare-metal.md [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+  daemonset.extensions/kube-flannel-ds-ppc64le created
+  daemonset.extensions/kube-flannel-ds-s390x created
+```
+
+## Join remaining nodes
+
+Run the above join commands on the nodes `kube02` and `kube03`. Below is the output on `kube02` for the setup for this guide:
+
+```
+kube02:~ > sudo kubeadm join 192.168.10.61:6443 --token blcr1y.49wloegyaugice8a --discovery-token-ca-cert-hash sha256:0505933664d28054a62298c68dc91e9b2b5cf01ecfa2228f3c8fa2412b7a78c8
```
Comment (lamai93) [inline docs/Manual/Tutorials/Kubernetes/bare-metal.md]: Placeholder for IP address?

Comment (kvahed) [reply docs/Manual/Tutorials/Kubernetes/bare-metal.md]: wrote the ip addresses into the introduction.

**Pull request merged**